From 8c0bc9e62e82674de80734cce8a83c020288fa5d Mon Sep 17 00:00:00 2001
From: Harsh J <harsh@apache.org>
Date: Fri, 25 May 2012 07:04:35 +0000
Subject: [PATCH 0139/1357] HDFS-2391. Newly set BalancerBandwidth value is not displayed anywhere. (harsh)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/trunk@1342520 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 3dde9ad1bb72db8b0659aaafc1a2fc28fe9fc118)
---
 .../hdfs/server/datanode/BPOfferService.java       |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/BPOfferService.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/BPOfferService.java
index f614e0b..7420684 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/BPOfferService.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/BPOfferService.java
@@ -609,6 +609,9 @@ class BPOfferService {
       if (bandwidth > 0) {
         DataXceiverServer dxcs =
                      (DataXceiverServer) dn.dataXceiverServer.getRunnable();
+        LOG.info("Updating balance throttler bandwidth from "
+            + dxcs.balanceThrottler.getBandwidth() + " bytes/s "
+            + "to: " + bandwidth + " bytes/s.");
         dxcs.balanceThrottler.setBandwidth(bandwidth);
       }
       break;
-- 
1.7.0.4

